app/controllers/posts_controller.rb

```diff
@@ -14,7 +14,7 @@ def new
   def show
     @user = User.find_by_id(params[:user_id])
     @post = Post.find_by_id(params[:id]) unless @user.nil?
-    @comments = @post.comments unless @post.nil?
+    @comments = @post.comments.includes(:author) unless @post.nil?
```

Comment:
- [ ] it would be better to do the same for `post` where the code should be `@user.posts.includes(:comments)` in order to complete the requirements